code/game/objects/items/weapons/tools/_tools.dm

```diff
@@ -758,7 +758,10 @@
 
 //Returns the amount of fuel in tool
 /obj/item/weapon/tool/proc/get_fuel()
-	return reagents.get_reagent_amount("fuel")
+	var/reagents_amount = 0
+	if(reagents)
+		reagents_amount = reagents.get_reagent_amount("fuel")
+	return reagents_amount
```

Comment:
I wonder if "return ( reagents ? reagents.get_reagent_amount("fuel") : 0 )" works better with it